integrations/langfuse/src/haystack_integrations/tracing/langfuse/tracer.py

```diff
@@ -121,12 +125,14 @@ def trace(self, operation_name: str, tags: Optional[Dict[str, Any]] = None) -> I
                 # Haystack returns one meta dict for each message, but the 'usage' value
                 # is always the same, let's just pick the first item
                 m = meta[0]
-                span._span.update(usage=m.get("usage"), model=m.get("model"))
+                usage = m.get("usage") if m.get("usage") else None
+                span._span.update(usage=usage, model=m.get("model"))
```

Comment:
These two lines do the same as the old line 124, what's the intention here?

Reply:
OpenAIGenerator is returning an empty dict when streaming is enabled. The langfuse connector throws a value error in that case. It can either be None or a valid usage dict. 

#797 
raise ValueError(
ValueError: Usage object must have either {input, output, total, unit} or {promptTokens, completionTokens, totalTokens}
Paris{'tracer': {'name': 'test_langfuse', 'trace_url': 'https://cloud.langfuse.com/trace/4f3dd665-52f1-4938-9882-680dfda94ee8'}, 'llm': {'replies': ['According to the given context, the capital of France is:\n\nParis'], 'meta': [{'model': 'gpt-3.5-turbo', 'index': 0, 'finish_reason': 'stop', 'usage': {}}]}}

Reply:
Got it, you can keep it one-line then: `span._span.update(usage=m.get("usage") or None, model=m.get("model"))`

Reply:
done